exercises/intergalactic-transmission/instructions.md

```diff
@@ -7,7 +7,7 @@ Your job is to help implement
 
 A parity bit is simple way of detecting transmission errors.
 The transmitters and receivers can only transmit and receive _exactly_ eight bits at a time (including the parity bit).
-The parity bit is set so that there is an _even_ number 1s in each transmission and is always the first bit from the right.
+The parity bit is set so that there is an _even_ number of 1s in each transmission, and the parity bit is always the first bit from the right.
```

Comment:
```suggestion
The parity bit is set so that there is an _even_ number of 1 bits in each transmission, and the parity bit is always the first bit from the right.
```